corehq/apps/consumer_user/tasks.py

```diff
@@ -0,0 +1,36 @@
+from celery.task import task
+from django.conf import settings
+from dimagi.utils.web import get_url_base
+from django.template.loader import render_to_string
+from django.utils.encoding import force_bytes
+from django.urls import reverse
+from corehq.apps.consumer_user.models import ConsumerUserInvitation
+from corehq.apps.hqwebapp.tasks import send_html_email_async
+from django.utils.http import urlsafe_base64_encode
+from django.core.signing import TimestampSigner
+from corehq.apps.hqcase.utils import update_case
+from django.utils.translation import ugettext_lazy as _
+from .const import CONSUMER_INVITATION_SENT
+from .const import CONSUMER_INVITATION_STATUS
+
+
+@task(serializer='pickle', queue=settings.CELERY_MAIN_QUEUE, ignore_result=True)
```

Comment:
Please remove `serializer='pickle'`. This is deprecated.
https://github.com/dimagi/commcare-hq/blob/master/docs/celery.rst#L31-L32

Please also remove the queue parameter, it is implied that this will run in the main queue. 

Why are you ignoring the result?

Reply:
Fixed it